### ISSUE TITLE
Expo av player transition not very smooth. 

### ISSUE THREAD

**Issue by FawadMahmood** [IjzerenHein/react-navigation-shared-element#96]

**Comment by FawadMahmood** [comment]
Is it possible to translate video player to next screen and keep it running. 

I am able to make it. But video not work as expected, I made a manual example using Animated api that works for me, but having issues with transition of the navigator. 

It will be great if I can make this library work for me for this task. Please help.

**Comment by drewandre** [comment]
@FawadMahmood could you provide a code example?

**Comment by drewandre** [comment]
@IjzerenHein I'm also curious how a video transition would behave. I got an example up and running locally, but I see the same issue as @FawadMahmood. Images and other views work great, but not only is the video animation a little laggy, the video pauses and resets itself. I tried seeking to where the video left off, but this isn't an ideal solution.

I think this is a unique situation for this library since all other transitions can unmount and remount themselves as a newly cloned component without flickering/issues. But videos would need to remain mounted and active during the transition. Do you think this is possible with this library?

Thanks!

**Comment by drewandre** [comment]
I've created [an example repo](https://github.com/drewandre/react-navigation-shared-element-issue-96-repro) to demonstrate this issue. It's a fork from the latest [react-navigation-shared-element-demo](https://github.com/IjzerenHein/react-navigation-shared-element-demo) repo.

Below is a gif showing how the video pauses. Ideally the `<Video />` component would remain mounted and would not pause during the transition before ultimately resetting once the transition has finished. Is this possible with this library, or even just using the core primitives from [react-native-shared-element](https://github.com/IjzerenHein/react-native-shared-element)?

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/13245041/94596769-72f94280-0241-11eb-8a02-cb351c2f2bd1.gif)

**Comment by FawadMahmood** [comment]
Yes, thank you for an example,  @drewandre I am facing exactly same issues, also performance is not very good.
